modules/common/route/route.go

```diff
@@ -37,10 +39,12 @@ func NewRoute(
 	route *routev1.Route,
 	labels map[string]string,
 	timeout time.Duration,
+	override *OverrideSpec,
```

Comment:
NewRoute is not used outside of lib-common so this parameter addition will not break the service operators.

Reply:
right. so far all operators just use the ExposeEndpoints, so NewRoute is kind of internal right now. but might change